easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java

```diff
@@ -65,21 +66,44 @@ public static boolean hasPermissions(Context context, String... perms) {
         return true;
     }
 
-
     /**
      * Request a set of permissions, showing rationale if the system requests it.
      *
-     * @param object Activity or Fragment requesting permissions. Should implement
-     *                {@link android.support.v4.app.ActivityCompat.OnRequestPermissionsResultCallback}
-     *                or
-     *                {@link android.support.v13.app.FragmentCompat.OnRequestPermissionsResultCallback}
-     * @param rationale a message explaining why the application needs this set of permissions, will
-     *                  be displayed if the user rejects the request the first time.
+     * @param object      Activity or Fragment requesting permissions. Should implement
+     *                    {@link android.support.v4.app.ActivityCompat.OnRequestPermissionsResultCallback}
+     *                    or
+     *                    {@link android.support.v13.app.FragmentCompat.OnRequestPermissionsResultCallback}
+     * @param rationale   a message explaining why the application needs this set of permissions, will
+     *                    be displayed if the user rejects the request the first time.
      * @param requestCode request code to track this request, must be < 256.
-     * @param perms a set of permissions to be requested.
+     * @param perms       a set of permissions to be requested.
      */
     public static void requestPermissions(final Object object, String rationale,
                                           final int requestCode, final String... perms) {
+        requestPermissions(object, rationale,
+                android.R.string.ok,
+                android.R.string.cancel,
+                requestCode, perms);
+    }
+
+    /**
+     * Request a set of permissions, showing rationale if the system requests it.
+     *
+     * @param object         Activity or Fragment requesting permissions. Should implement
+     *                       {@link android.support.v4.app.ActivityCompat.OnRequestPermissionsResultCallback}
+     *                       or
+     *                       {@link android.support.v13.app.FragmentCompat.OnRequestPermissionsResultCallback}
+     * @param rationale      a message explaining why the application needs this set of permissions, will
+     *                       be displayed if the user rejects the request the first time.
+     * @param positiveButton custom text for positive button
+     * @param negativeButton custom text for negative button
+     * @param requestCode    request code to track this request, must be < 256.
+     * @param perms          a set of permissions to be requested.
+     */
+    public static void requestPermissions(final Object object, String rationale,
+                                          @StringRes int positiveButton,
```

Comment:
For consistency, could we do String arguments instead of resource ids here to match how `rationale` is passed?